codespell_lib/tests/test_dictionary.py

```diff
@@ -175,6 +175,8 @@ def test_error_checking_in_aspell(err, rep, err_aspell, rep_aspell, match):
 allowed_dups = {
     ('dictionary.txt', 'dictionary_en-GB_to_en-US.txt'),
     ('dictionary.txt', 'dictionary_rare.txt'),
+    ('dictionary.txt', 'dictionary_usage.txt'),
+    ('dictionary_rare.txt', 'dictionary_usage.txt'),
```

Comment:
I don't think this one is required now is it since you removed the terms from there?

Reply:
Removing the first one would trigger:
```
E                       AssertionError: error blaclist: correction blacklist from dictionary dictionary.txt is an error itself in dictionary dictionary_usage.txt
E                       assert 'blacklist' not in {'blacklist': ['blocklist'], 'blacklists': ['blocklists'], 'master': ['primary', 'leader', 'active', 'writer', 'coordinator', 'parent'], 'masters': ['primaries', 'leaders', 'actives', 'writers', 'coordinators', 'parents'], ...}
```

Removing the latter would trigger:
```
E                       AssertionError: error mater: correction master from dictionary dictionary_rare.txt is an error itself in dictionary dictionary_usage.txt
E                       assert 'master' not in {'blacklist': ['blocklist'], 'blacklists': ['blocklists'], 'master': ['primary', 'leader', 'active', 'writer', 'coordinator', 'parent'], 'masters': ['primaries', 'leaders', 'actives', 'writers', 'coordinators', 'parents'], ...}
```

Reply:
Makes sense, I'd forgotten about mater.